noxfile.py

```diff
@@ -53,14 +53,16 @@ def docs(session: nox.Session) -> None:
 
     session.install("-r", "docs/requirements.txt")
     session.chdir("docs")
+    if "pdf" in session.posargs:
+        session.run("sphinx-build", "-M", "latexpdf", ".", "_build")
+        return
     session.run("sphinx-build", "-M", "html", ".", "_build")
 
-    if session.posargs:
-        if "serve" in session.posargs:
-            print("Launching docs at http://localhost:8000/ - use Ctrl-C to quit")
-            session.run("python", "-m", "http.server", "8000", "-d", "_build/html")
-        else:
-            print("Unsupported argument to docs")
+    if "serve" in session.posargs:
+        print("Launching docs at http://localhost:8000/ - use Ctrl-C to quit")
+        session.run("python", "-m", "http.server", "8000", "-d", "_build/html")
+    elif session.posargs:
+        print("Unsupported argument to docs")
```

Comment:
Should this error / exit?

Reply:
Probably, wasn't sure how to do that. Guess I'll look it up... Ah, there's a method for it, that's easy.